pipeline/outputs/datadog.md

```diff
@@ -11,6 +11,7 @@ Before you begin, you need a [Datadog account](https://app.datadoghq.com/signup)
 | Host | _Required_ - The Datadog server where you are sending your logs. | `http-intake.logs.datadoghq.com` |
 | TLS | _Required_ - End-to-end security communications security protocol. Datadog recommends setting this to `on`. | `off` |
 | compress | _Recommended_ - compresses the payload in GZIP format, Datadog supports and recommends setting this to `gzip`. |  |
+| Proxy _Optional_ | Specify an HTTP Proxy. The expected format of this value is _http://host:port_. Note that _https_ is __not__ supported yet. |  |
```

Comment:
the word _Optional_ should not be there, it would lead to confusion since that column is just for the property name

Reply:
Fixed!